.github/pull_request_template.md

```diff
@@ -1,9 +1,15 @@
-**Description:** <Describe what has changed.>
 <!--Ex. Fixing a bug - Describe the bug and how this fixes the issue.
 Ex. Adding a feature - Explain what this achieves.-->
+#### Description
 
-**Link to tracking Issue:** <Issue number if applicable>
+<!-- Issue number if applicable -->
+#### Link to tracking issue
+Fixes #
```

Comment:
```suggestion
Fixes
```
Nit: When pasting the full issue URL, it gets displayed as `#12345`, with the `#` included. I often end up having to go back and delete the additional `#` manually. This might be just be a "me" problem though, definitely not a blocker.

Reply:
I was thinking about that and couldn't really come up with a good solution. I was hoping that including the `#` would nudge users to just paste the issue number without thinking about it too hard, but personally I also paste the whole URL.

I'll remove it for now and update the docs to be a bit more descriptive about what to put here. If we run into problems with issues being linked, we can update this in the future.